scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
```

Comment:
see line 41. i need more than just `0.4.3` because this is not a release type of build, it's the rolling unstable script.